artifactory/commands/buildinfo/publish.go

```diff
@@ -70,6 +70,10 @@ func (bpc *BuildPublishCommand) IsDetailedSummary() bool {
 }
 
 func (bpc *BuildPublishCommand) CommandName() string {
+	isAutoPublishedTriggered, err := clientutils.GetBoolEnvValue(coreutils.UsageAutoPublishedBuild, false)
+	if err == nil && isAutoPublishedTriggered {
```

Comment:
If the error isn't nil, let's log the error, with a prefix that mention the context of the error.
The same applies for the `CommandName` function in the `config.go` file.